python/fastexcel/__init__.py

```diff
@@ -171,8 +173,10 @@ def load_sheet_by_idx(
         :param schema_sample_rows: Specifies how many rows should be used to determine
                                    the dtype of a column.
                                    If `None`, all rows will be used.
-        :param use_columns: Specifies the columns to use. Can either be a list of column names, or
-                            a list of column indices (starting at 0).
+        :param use_columns: Specifies the columns to use. Can either be a list of column names, a
+                            a list of column indices (starting at 0), or a list of columns and
+                            ranges as column names indices in excel form. For example, `"A,C:E,F"`
```

Comment:
no it should be inclusive both ways as an excel user

Reply:
So `A,C,D,E,F` ?

Reply:
exactly

Reply:
248acdd